wiki/Tournaments/Official_support/en.md

```diff
@@ -161,23 +176,32 @@ Badge images that do not meet these standards will be refused with given reasoni
 
 In exceptionally rare cases, a tournament may be granted the privilege of awarding badges to its top three finalists. A tournament that has received this distinction is recognised for attaining the highest level of production quality and organisation possible — this is given exclusively at the discretion of the osu! team. If you believe your tournament meets the expectations to receive this distinction, you may request it when submitting your initial badge request.
 
+Note: podium finishing/finalist badge prizes are only awarded if the tournament in question does not have any restrictions in place for participation, such as being limited to a specific rank range or a specific geographic region.
+
 Examples of exceptional tournaments that have met these expectations:
 
 - [Corsace Open](https://osu.ppy.sh/community/forums/topics/1607468)
 - [Expert Global Taiko Showdown](/wiki/Tournaments/GTS/EGTS_2022)
 - [Springtime osu!mania Free-for-all Tournament](/wiki/Tournaments/SOFT/6)
 
-### Requesting main-menu banner support
+### Requesting extra visibility support (main-menu banners and news posts)
```

Comment:
what kind of term is this

```suggestion
### Requesting advertising support
```

Reply:
i'm not sure advertising is the right term for it, hence the hesitation

(that and it is the Forbidden Word)

Reply:
can you come up with some other euphemism because "extra visibility support" sounds stupid

Reply:
outreach? engagement?

Reply:
for now this works ok i think

```suggestion
### Requesting in-game banner and news post support
```

or even this for when the website news carousel starts becoming a thing

```suggestion
### Requesting in-game and website outreach support
```